hw/ip/prim/prim_flash.core

```diff
@@ -10,6 +10,7 @@ filesets:
     depend:
       - lowrisc:prim:prim_pkg
       - lowrisc:prim:primgen
+      - lowrisc:prim:ram_1p
```

Comment:
This is already added to `prim_generic_flash.core`. However, the generator for the ram1p does not kick in, causing compile errors.

Reply:
Can you add this comment to the code?